lib/vanagon/engine/pooler.rb

```diff
@@ -36,12 +37,17 @@ def load_token
       end
 
       def token_from_file
-        token_file = File.expand_path("~/.vanagon-token")
+        vanagon_token_file = File.expand_path("~/.vanagon-token")
+        vmfloaty_token_file = File.expand_path("~/.vmfloaty.yml")
```

Comment:
Conversation, not criticism: should we consider updating these to use `Dir.home` instead of `~`?

```ruby
vanagon_token_file = File.join(Dir.home, (".vanagon-token"))
vmfloaty_token_file = File.join(Dir.home, (".vmfloaty.yml"))
```

Thinking about it, `~` is a UNIX convention that most stuff *probably* understands, but `Dir.home` is pretty explicit. Both respect the value of `ENV['HOME']` but should we stop making UNIX-first assumptions about our tools?